src/Common/src/Common.Security/ConfigureNamedCertificateOptions.cs

```diff
@@ -10,24 +10,29 @@
 
 namespace Steeltoe.Common.Security;
 
-public class ConfigureCertificateOptions : IConfigureNamedOptions<CertificateOptions>
+public sealed class ConfigureNamedCertificateOptions : IConfigureNamedOptions<CertificateOptions>
```

Comment:
Poor naming and different behavior. When there are two classes, those not prefixed with `Pem` are for single-file scenarios like PKCS#12. In the single-file path, the certificate is loaded here rather than at the IConfiguration level.